in_toto/verifylib.py

```diff
@@ -1311,11 +1311,11 @@ def verify_sublayouts(layout, chain_link_dict, superlayout_link_dir_path):
         # name relative the the current link directory path, i.e. if there
         # are multiple levels of sublayout nesting, the links are expected to
         # be nested accordingly
-        sublayout_link_dir = SUBLAYOUT_LINK_DIR_FORMAT.format(
-            name=step_name, keyid=keyid)
+        sub_link_dir = in_toto.models.layout.SUBLAYOUT_LINK_DIR_FORMAT.format(
```

Comment:
This is what i mean.. I am not sure if `sub_link_dir` is okay as variable name... However we only use it in the next line. What do you think?